Pixel Game/Assets/Scripts/InventorySlot.cs

```diff
@@ -18,7 +18,7 @@ public class InventorySlot : MonoBehaviour
     private GameObject playerCharacter;
     private Color ColorOfSlot;
     private Vector2 itemDropPosition;
-    private ItemData ItemDataInSlot;
+    public ItemData ItemDataInSlot; // Changed from private to public. All good?

```

Comment:
=> is a arrow function right? I dont get how it works with just ```public ItemData ItemDataInSlot => itemDataInSlot;```
Care to show me tomorrow?

Reply:
=> is in this case is a one-line shorthand for
```c#
public ItemData ItemDataInSlot
{
    get
    {
        return itemDataInSlot;
    }
}
```

Reply:
Aaah very nice! Didnt know that, so its just a public get function. Shouldnt it be named GetItemDataInSlot or something then?

Reply:
The standard way of writing in C# is actually to not use Get in the name, because you access it like a normal member variable without ().

Reply:
For a normie like me this does not seem very easy to read.

Reply:
But it's still hinted in the name that it is a property and not a field because you're supposed to always start property and function names with capital letters.